docs/HowToUseIotivity.md

```diff
@@ -0,0 +1,12 @@
+How to use IoTivity
+===================
+
+IoTivity is an open source software framework enabling seamless device-to-device connectivity to address the emerging needs of the Internet of Things.
+If you want to know more detailed information, please find it at [README](../external/iotivity/iotivity_1.2-rel/README.md) and https://iotivity.org
+
+----------
+##To use the IoTivity feature
```

Comment:
1. Add a space
2. Here is only how to use pre-set config. When someone wants to enable iotivity from own config, he can't know how to do it.

Reply:
I see